src/components/ListItem/ListItem.tsx

```diff
@@ -108,10 +111,13 @@ const ListItem: FC<ListItemProps> & { sizes?: typeof SIZES } = forwardRef(
 
     return (
       <div
+        data-testid={dataTestId || getTestId(ComponentDefaultTestId.LIST_ITEM, id)}
         ref={mergedRef}
-        className={cx("list-item", className, BEMHelper(size.toString()), {
+        className={cx(styles.listItem, "list-item", className, BEMHelper(size.toString()), {
           [BEMHelper("selected")]: selected && !disabled,
-          [BEMHelper("disabled")]: disabled
+          [styles.selected]: selected && !disabled,
+          [BEMHelper("disabled")]: disabled,
+          [styles.disabled]: disabled
```

Comment:
Let't change the order - first modular className then global

Reply:
I am not sure how it's better

Reply:
First of all we do like this everywhere, so it's just conventional, at second, I still guess that this means that modular styles would win global in terms of specificity...